test_requirements.txt

```diff
@@ -32,16 +29,14 @@ packaging==21.3
 pluggy==0.13.1
     # via pytest
 py==1.11.0
-    # via
-    #   pytest
-    #   pytest-forked
+    # via pytest-forked
 pylint==2.7.1
     # via -r test_requirements.in
 pylintfileheader==0.3.0
     # via -r test_requirements.in
 pyparsing==3.0.9
     # via packaging
-pytest==6.1.2
+pytest==7.2.1
```

Comment:
Pytest needed to be upgraded to correct this error in python 3.10 `TypeError: required field "lineno" missing from alias`.